### DISCUSSION TITLE
eqzumer evolution firmware with FM , AM and DSB thanks to sfotis analized on Youtube

### DISCUSSION THREAD

**Discussion by Hunterfnet** [egzumer/uv-k5-firmware-custom#398] [General]
https://www.youtube.com/watch?v=kKCTYIddeCg
First at all thanks to [sfotis](https://github.com/sfotis) for the version this post is for mention his work. 
Congratulate sfotis for his experimental work , and finally his work has fruits in FM, AM and USB transmission as seen in the YouTube video. Now possibly IJV has a rival, maybe someone wants to continue with the project adjusting reception, real BF's  and RF's filters and the parts that fail and/or are missing on the screen. Congratulations.

**Comment by ErikS-web** [comment]
I think you post this on wrong place.

Here we **_enjoy_** the firmware from **EGZUMER** :+1:

**Reply by gartnl** [reply]
Just ignore it. Imo it's just trolling. 
And there is no USB TX. There is a crappy DSB TX that sounds horrible. No HAM would ever regard that as a serious option.
And I've yet to see what a SA makes of it, I bet it is a disaster. Real filters are just a dream but simply not possible.
Sure, it is fun to experiment with the BK chip. But the enhancements egzumer made on the UI are far more important. 

I really don't get why people expect their 17 dollar HT to be a $3000  all-band all-mode HF/VHF/UHF set.

**Comment by Hunterfnet** [comment]
I don't care what you say Erik . I think too you are out of orbit and place  . Don't care , i'm using other firmware because i dont see any evolution for weeks here with a lot of failures unsolved , bugs ,etc and no action only bla bla bla , i think the main function of a technical channel is solve the main real mistakes not be a sect of profetic adulators  club . For my part it's my last post here . I posted here because is an egzumer version and was not possible post on sfotis place . Its all , you can continue making the butler with no ideas and have good luck out here ;). Bye and good work guys ,

**Comment by majonezz** [comment]
> I don't care what you say Erik . I think too you are out of orbit and place . Don't care , i'm using other firmware because i dont see any evolution for weeks here with a lot of failures unsolved , bugs ,etc and no action only bla bla bla , i think the main function of a technical channel is solve the main real mistakes not be a sect of profetic adulators club . For my part it's my last post here . I posted here because is an egzumer version and was not possible post on sfotis place . Its all , you can continue making the butler with no ideas and have good luck out here ;). Bye and good work guys ,

WTF are you talking about? You wanna motivate them that way? If you want to contribute, have skills, then go ahead. If you want to use other firware, why not? I think i don't understand that kind of attitude. You demand something from guys that doing their best in their free time, for no profit. Come on.

**Reply by lohtse** [reply]
The funny part is when his new love fw  stops updating(no Updates in over a month for it  BTW) and developing further etc he will comment using the same BS.. Hunterfnet
 and POS's always do as they are never happy and always want more and more.

**Comment by CrStuff** [comment]
Some guys need accept that exist people that like drive not defective cars and don't like buy TESLA's or IPHONE's .
Some guys need see and hear the correct function and feel a working software with not fake filters or simply don't like drive a car with no brake lights or an unfinished control panel . All is more easy that loving a defective car or phone or simply an unfinished project.
I know too that the toys market exist for one reason.
